### DISCUSSION TITLE
New Feature request Space Weather alerts

### DISCUSSION THREAD

**Discussion by creinemann** [JVital2013/vitality-goes#41] [Ideas]
Since we are peaking in the solar cycle, perhaps add an alert similar to the weather alerts, but scanning the Space Weather Messages for Codes : such as WARSUD, WATA50, WATA99, ALTK08,ALTK09,WARK06,WARK07,WARK09

You can find a list of the message code here: https://www.swpc.noaa.gov/content/subscription-services

Carl

**Comment by JVital2013** [comment]
I like this idea! It could probably be added into the existing alerts system without too much trouble. I'm working on some other changes to EMWIN data loading, and I'll take a stab at adding this.

**Comment by JVital2013** [comment]
@creinemann I'm implementing WATA50, WATA99, ALTK07, ALTK08, and ALTK09. It will be in Vitality GOES v1.5, to be released later this week. It will be an option that can be turned on (off by default). These messages are already on the Other EMWIN page by default (which again, can be customized starting in 1.5)

The WAR* variants are not transmitted over EMWIN - at least, not in the last 1.5 years. I verified the data isn't "hidden" somewhere - Three  WAR* files were issued on June 24, but their text is found nowhere in the EMWIN data stream.

![image](https://github.com/JVital2013/vitality-goes/assets/24253715/65c2aa31-73db-4a5f-9fc3-e95f079969a0)

**Comment by JVital2013** [comment]
Implemented in 1.5

**Reply by creinemann** [reply]
Hurray!
